Cargo.toml

```diff
@@ -19,7 +19,10 @@ bitflags = "1.0.1"
 serde = { version = "1.0.34", features = ["derive"] }
 serde_json = "1.0.50"
 serde_repr = "0.1"
-url = {version = "2.0.0", features = ["serde"]}
+# https://github.com/yescallop/fluent-uri-rs/pull/9
+# https://github.com/yescallop/fluent-uri-rs/pull/10
+fluent-uri = { git = "https://github.com/gibbz00/fluent-uri-rs", branch = "mixed_features"}
+derive_more = "0.99"
```

Comment:
Adding this crate for two simple derives doesn't seem worth it, just manually implement them instead.

Reply:
Fair! I do love the `derive_more` crate, but you're in right that this isn't the time to introduce it.